lib/plugins/features/movies/controller.js

```diff
@@ -4,5 +4,27 @@ const Movie = require('../../../models/movie');
 
 exports.create = (payload) => {
   return new Movie().save(payload)
-  .then((movie) => new Movie({ id: movie.id }).fetch());
+    .then((movie) => new Movie({ id: movie.id }).fetch());
+};
+
+exports.findAll = (filter) => {
+  return new Movie().query((qb) => {
+    if (filter.release_year) {
+      qb.where({ release_year: filter.release_year });
+    }
+    if (filter.released_after) {
+      qb.where('release_year', '>', filter.released_after);
+    }
+    if (filter.released_before) {
+      qb.where('release_year', '<', filter.released_before);
+    }
```

Comment:
an fyi about the convention for equality predicate filters in lob-api: we usually use an object to pass these in, like this:
```
date_created: {
  gt: ,
  lt: ,
  lte: ,
  eq: ,
  ...
}
```
what you have is also a totally logical way of doing the before and after filters, no need to change it here. just letting you know that you will see something different when working on api!

here are a couple examples from our codebase:

https://github.com/lob/validators/blob/012855db229a5320c8bbb104c24e33409be5b735/lib/date-filter.js#L9-L14

https://github.com/lob/lob-api/blob/297915175943854248c3775e09c6ac5e0032ae8d/src/validators/checks/list.js#L101-L106